Server/Plugins/APIDump/Classes/BlockEntities.lua

```diff
@@ -1365,6 +1365,22 @@ World:ForEachChestInChunk(Player:GetChunkX(), Player:GetChunkZ(),
 				},
 				Notes = "Returns the amount of the nearby players in a 16-block radius.",
 			},
+			GetSpawnCount =
+			{
+				Returns =
+				{
+					Type = "number",
+				},
+				Notes = "Returns the number of entities the spawner will try to spawn on each activation.",
+			},
+			GetSpawnRange =
+			{
+				Returns =
+				{
+					Type = "number",
+				},
+				Notes = "Returns the shortest radius of the square the spawner tries to spawn entities in.",
```

Comment:
A square cannot have a 'shortest radius'???

Reply:
It doesn't have a normal radius eiter...

Reply:
"half the lenght of the square"

Reply:
That could work.